core-services/importExport/src/main/java/org/silverpeas/core/importexport/model/RepositoryType.java

```diff
@@ -29,40 +29,42 @@
  */
 package org.silverpeas.core.importexport.model;
 
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlRootElement;
+
 /**
  * @author tleroi To change the template for this generated type comment go to
  * Window&gt;Preferences&gt;Java&gt;Code Generation&gt;Code and Comments
  */
+@XmlRootElement
+@XmlAccessorType(XmlAccessType.NONE)
 public class RepositoryType {
 
-  public final static int NO_RECURSIVE = 0;
-  public final static int RECURSIVE_NOREPLICATE = 1;
-  public final static int RECURSIVE_REPLICATE = 2;
-  public final static String NO_RECURSIVE_STRING = "NO_RECURSIVE";
-  public final static String RECURSIVE_NOREPLICATE_STRING = "RECURSIVE_NOREPLICATE";
-  public final static String RECURSIVE_REPLICATE_STRING = "RECURSIVE_REPLICATE";
+  public static final int NO_RECURSIVE = 0;
+  public static final int RECURSIVE_NOREPLICATE = 1;
+  public static final int RECURSIVE_REPLICATE = 2;
+  private static final String NO_RECURSIVE_STRING = "NO_RECURSIVE";
+  private static final String RECURSIVE_NOREPLICATE_STRING = "RECURSIVE_NOREPLICATE";
+  private static final String RECURSIVE_REPLICATE_STRING = "RECURSIVE_REPLICATE";
+
+  @XmlAttribute
   private String path;
+  @XmlAttribute
   private String componentId;
+  @XmlAttribute
   private int topicId;
+  @XmlAttribute
   private String massiveType;
 
-  /**
-   * @return
-   */
-  public String getComponentId() {
-    return componentId;
+  public RepositoryType() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1186)